avro-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/AWSKafkaAvroConverter.java

```diff
@@ -44,6 +47,8 @@
 @Slf4j
 @Data
 public class AWSKafkaAvroConverter implements Converter {
+    private static final String STS_ROLE_ARN = "sts.roleArn";
```

Comment:
Could you please replace the naming from camel casing to dotted.
e.g. "sts.role.arn"

We have learned that some of the Docker conversions assume Kafka standard of having no camel casing. Some external Docker images provide configurations as ALL CAPS with UNDERSCORES and then translate _ to . and CAPS to small.